examples/selective_run/README.md

```diff
@@ -0,0 +1,21 @@
+# Selective run
+This example demonstrates the ability to execute only parts of a pipeline
+using the `-i` command and explicit step/service selection.
+
+The structure of the example is visualized in [pipeline.svg](./pipeline.svg).
+
+The pipeline depicts how an active service could be updated with minimal
+downtime and validation of a new version.
+
+To rerun the validation without preparing a new version
+```
+gantry -i prepare_new_service_version test_new_service
+```
+can be used. This forces gantry to ignore all non-dependencies of
+`test_new_service`. `prepare_new_service_version` is explicitly ignored thus
+`otherthing` and `something` are not considered dependencies for
+`test_new_service` and are ignored.
```

Comment:
I would rename `otherthing` and `something` to make it more clear that they are prerequisites for `prepare_new_service_version` maybe `pre_prepare_0` and `pre_prepare_1`